tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+envlist = py{27,35}-django{18,19}, quality
+
+[testenv]
+setenv =
+    # This allows us to reference test_settings.py
+    PYTHONPATH = {toxinidir}
+
+deps =
+    django18: Django>=1.8,<1.9
+    django19: Django>=1.9,<1.10
+    -rtest_requirements.txt
+
+commands =
+    django-admin.py test edx_rest_framework_extensions --settings=test_settings --with-coverage --cover-package=edx_rest_framework_extensions
+    coverage report
```

Comment:
@clintonb Yeah, you can't print a coverage report without first running the test.
My approach is basically:
- _only_ run tests in the default testenvs (`py{27,35}-django{18,19}`)
    - do not generate coverage stats or print a report
- add a separate testenv called `coverage` that runs tests and generates coverage

Here's what I have in another project.
I tried to remove unrelated work and add comments inline.

Hopefully that helps.

# tox.ini

```ini
# You can also add other tests here like `pylint`, `pep8`, etc.
[tox]
envlist = py27-dj18,coverage

# By default, just run tests normally.
# This will not generate coverage stats, nor will it generate a report.
[testenv]
commands = {envpython} manage.py test

# Typically, just run this locally.
# This should test code, generate coverage stats, and print a report.
[testenv:coverage]
deps =
    coverage
setenv =
    NOSE_COVER_TESTS=1
    NOSE_WITH_COVERAGE=1
commands =
    {envpython} manage.py test

# Only run this on Travis
# This should test code, generate coverage stats, and submit to coveralls.io
[testenv:coveralls]
deps =
    coverage
    coveralls
setenv =
    NOSE_COVER_TESTS=1
    NOSE_WITH_COVERAGE=1
passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
commands =
    {envbindir}/coverage run --source=geoinfo manage.py test
    {envbindir}/coveralls
```

# .travis.yml

```yaml
script:
    - 'tox -v'
env:
    - TOXENV=py27-dj18
    - TOXENV=coveralls
```

# Makefile
```makefile
# The default `envlist` only contains `coverage`, not `coveralls`.
test:
    python manage.py test
```